### DISCUSSION TITLE
cannot touch/cannot access errors

### DISCUSSION THREAD

**Discussion by stayupthetree** [RandomNinjaAtk/arr-scripts#224] [Q&A] [answered]
I seem to have a bunch of these in my Sonarr logs, though everything appears to be working, is this something I should be concerned or fix?

[Error] DailySeriesEpisodeTrimmer.bash: jq: error: writing output failed: Broken pipe 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/1054': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/1054': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/80748': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/80748': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/125282': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/125282': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/71712': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/71712': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/94372': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/94372': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/208569': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/208569': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/79744': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/79744': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/121658': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/121658': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/202308': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/202308': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/2051': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/2051': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/125282': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/125282': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/202308': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/202308': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/84958': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/84958': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/80748': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/80748': Permission denied 
[Error] Extras.bash: touch: cannot touch '/config/extended/logs/extras/71712': Permission denied 
[Error] Extras.bash: chmod: cannot access '/config/extended/logs/extras/71712': Permission denied

**Comment by RandomNinjaAtk** [comment]
Seems like you have a file permissions issue, you can correct the file permissions via command line and that will resolve your problem

**Reply by stayupthetree** [reply]
I chmod 777 everything and the error always comes back, The folders are owned by nobody:users as well. It doesn't happen 100% of the time, as there are tons of files/folders in this location successfully created.

**Reply by RandomNinjaAtk** [reply]
Did you run the permissions change recurrsively?

**Reply by stayupthetree** [reply]
Yes

**Reply by RandomNinjaAtk** [reply, marked as answer]
Then IDK, its not reproducible in my configuration. Maybe you have something else adjusting the file permissions that is causing it... 

You should double check the permissions are what you set them too. Because 777 would give everyone read/write/execute access....
